### DISCUSSION TITLE
how-to-set-default-audio-in-ubuntu

### DISCUSSION THREAD

**Discussion by giscus** [thomasnotfound/how.wtf#149] [Blog Posts]
# how-to-set-default-audio-in-ubuntu

How to set default audio in Ubuntu on startup using pactl

https://how.wtf/how-to-set-default-audio-in-ubuntu.html

**Comment by cpbl** [comment]
output is "source" and input is "sink"?   Might you have this  backwards?

**Reply by thomasgtaylor** [reply]
Oops! I mixed up the order in the last two paragraphs. You are correct - I will adjust. Thank you!
